spec/request/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+describe 'DELETE api/v1/targets/1', type: :request do
+  let!(:user) { create(:user, :confirmed) }
+  let!(:auth_header) { user.create_new_auth_token }
+  let!(:target) { create(:target, user: user) }
+
+  context 'when the request is succesful' do
+    subject { delete api_v1_target_path(target), headers: auth_header }
+    it 'is expected a successful response' do
+      subject
+      expect(response).to have_http_status(:no_content)
+    end
+
+    it 'is expected a decrement of targets count by 1' do
+      expect { subject }.to change(Target, :count).by(-1)
+    end
+  end
+
+  context 'when the target not exist to the same user' do
```

Comment:
This context is not readable enough. Try something like `when the target wasn't created by user` or `when the target doesn't exist`